src/testdir/test_popup.vim

```diff
@@ -896,4 +896,48 @@ func Test_menu_only_exists_in_terminal()
   endtry
 endfunc
 
+func Test_popup_complete_mode()
+  new
+  inoremap <f5> <c-r>=complete_mode()<cr>
```

Comment:
The test has a side-effect: it leaks this map. We should do iunmap <F5> before exiting the test.

Reply:
Or put `<buffer>`.

Reply:
Oh, it seems better.